man/systemd-socket-proxyd.xml

```diff
@@ -115,6 +122,9 @@ server {
         <programlisting><![CDATA[# systemctl enable --now proxy-to-nginx.socket
 $ curl http://localhost:80/]]></programlisting>
       </example>
+      <para>If <filename>nginx.service</filename> has <varname>StopWhenUnneeded=</varname> set, then
+      passing <option>--exit-idle-time</option> to <command>systemd-socket-proxyd</command> allows
```

Comment:
Since the argument is always required, add "=": `<option>--exit-idle-time=</option>`.